packages/web-components/docs/api-report.md

```diff
@@ -1126,6 +1127,9 @@ export enum PaletteType {
     neutral = "neutral"
 }
 
+// @public
+export function parseColorString(color: string): ColorRGBA64;

```

Comment:
Will want to review this change with @chrisdholt this is the web-component implementation of Fluent UI

Reply:
I don't know why this change was included, or why the api files changed; the changes don't look caused by any of the code I wrote, I'm looking into what's going on with those now :)

Reply:
Yeah this is odd as the API report will typically cause the build to fail, thus, not allowing something to be checked-in. From what I can tell, all of our previous PR's passed the build so it's odd this is being generated. Not sure when you branched for this work but you could try rebasing w/ upstream to see if the report somehow fixed itself?

I'm fine if this change is included (it may have to be or it could prevent the build from passing). If it happens again we'll want to keep an eye on it. The only ask I would have would be to update the change request for it to "none" so that the packages do not update.

Reply:
I think the API Extractor reports update if you run a full local build. That's probably how this got in.

Reply:
Oh another possibility is that maybe if you had the latest code, but didn't update dependencies (`yarn`) before doing a full local build, that could have caused a bad update.

Reply:
Thanks so much, updating + rebuilding seems to have fixed it. I'm going to double check, then update :)